jupyter/dev-entrypoint.bash

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
 cd /pymor
```

Comment:
I believe this line may also have to be removed, since the earlier docker image also complained that it couldnot find this path within the container.

Reply:
Since I'm probably assuming to be in the mounted `/pymor` source directory somewhere downstream, I've made this `cd` conditional on the dir actually existing.